### ISSUE TITLE
Next Page button

### ISSUE THREAD

**Issue by orhankutlu** [balderdashy/sails-docs#702]
It is not a big deal but it would be nice to have "Next Page" button at the sails documentation web page. It makes navigation between related documents easier. Would be good for beginners.

**Comment by sailsbot** [comment]
@orhankutlu: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!
